### PR TITLE
[HSEARCH] 7.2.0.Alpha1

### PR DESCRIPTION
**Pull request by marko-bekhta**

### CODE REVIEW

File: posts/MarkoB/2024-05-16-hibernate-search-7-2-0-Alpha1.adoc [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+        .where( f -> f.queryString()
+                .field( "numberOfPages" )
+                .matching( "[350 TO 800]" )
+        )
+        .fetchHits( 20 );
+----
+
+See corresponding sections in link:{hsearch-doc-url-prefix}#search-dsl-predicate-simpleQueryString-format-types[`simpleQueryString`]
+link:{hsearch-doc-url-prefix}#search-dsl-predicate-query-string-format-types[`queryString`] documentation
+for any limitations.
```
Comment (yrodiere) [inline posts/MarkoB/2024-05-16-hibernate-search-7-2-0-Alpha1.adoc]:
```suggestion
for details.
```

**Review by yrodiere — APPROVED** [review]
LGTM, just one suggestion.

File: posts/MarkoB/2024-05-16-hibernate-search-7-2-0-Alpha1.adoc [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+[source, JAVA, indent=0, subs="+callouts"]
+----
+List<Book> hits = searchSession.search( Book.class )
+        .where( f -> f.withParameters( params -> f.bool() // <1>
+                .should( f.match().field( "title" )
+                        .matching( params.get( "title-param", String.class ) ) ) // <2>
+                .filter( f.match().field( "genre" )
+                        .matching( params.get( "genre-param", Genre.class ) ) ) // <3>
+        ) )
+        .param( "title-param", "robot" ) // <4>
```
Comment (gsmet) [inline posts/MarkoB/2024-05-16-hibernate-search-7-2-0-Alpha1.adoc]: It's a bit hard to see the value with this example? Maybe you should store the query and show how to execute it twice with different parameters?

**Review by gsmet — COMMENTED** [review]
Nice iteration, I added a very minor comment.

File: posts/MarkoB/2024-05-16-hibernate-search-7-2-0-Alpha1.adoc (hunk 1 above)
Comment (marko-bekhta) [reply posts/MarkoB/2024-05-16-hibernate-search-7-2-0-Alpha1.adoc]: Thanks for taking a look! 😃

File: posts/MarkoB/2024-05-16-hibernate-search-7-2-0-Alpha1.adoc (hunk 2 above)
Comment (marko-bekhta) [reply posts/MarkoB/2024-05-16-hibernate-search-7-2-0-Alpha1.adoc]: Thanks for the suggestion! would this example be better?

Comment (gsmet) [reply posts/MarkoB/2024-05-16-hibernate-search-7-2-0-Alpha1.adoc]: Yup, perfect!

**Pull request merged**